bazel/repositories.bzl

```diff
@@ -70,6 +70,10 @@ def envoy_gloo_dependencies():
         # revert upstream's cache filter from using a separate HttpAsyncClient
         "@envoy_gloo//bazel/foreign_cc:002-revert-cache_filter-separate-upstream.patch",
         "@envoy_gloo//bazel/foreign_cc:004-fixed-upstream-filter-disabled-by-default.patch",
+        # use nghttp2 as the default http/2 implementation over oghttp2. see
+        # the slack context in
```

Comment:
nit: you say "slack context" but link a github issue, and the issue doesn't have any slack links. In the future, I think we can remove slack. For this, im happy to merge as is....perhaps we link the slack context in that github issue?